tests/api_fastapi/core_api/routes/public/test_connections.py

```diff
@@ -522,8 +522,8 @@ def test_patch_should_respond_200(self, test_client, body, session):
                     "conn_type": TEST_CONN_TYPE,
                     "extra": None,
                     "host": TEST_CONN_HOST,
-                    "login": None,
-                    "port": None,
+                    "login": TEST_CONN_LOGIN,
+                    "port": TEST_CONN_PORT,
```

Comment:
Why was this test updated ? It's still valid to actually send `None` to the API.

Reply:
Actually it's not. I will pass the serializer and then crash on the db side trying to insert non nullable values. 🤔

Reply:
We have `login` and `port` in the `update_mask`, but neither of them is included in the request body. As a result, when we intersect `fields_to_update` with `update_mask`, there will be no common fields. Consequently, nothing will be updated, and the value will remain unchanged instead of being set to `None`.

Reply:
It dont give 500 error, its just assert failed